connection.go

```diff
@@ -488,7 +488,10 @@ func (c *Connection) dispatch0(f frame) {
 				c <- Blocking{Active: false}
 			}
 		default:
-			c.rpc <- m
+			select {
+			case c.rpc <- m:
+			default:
```

Comment:
Why does this now drop if there is no longer a receiver on the `rpc` buffered channel?